std/numeric.d

```diff
@@ -1647,7 +1647,9 @@ unittest
 }
 
 /**
-Computes accurate sum of binary logarithms of input range $(D r).
+Computes sum of binary logarithms of input range $(D r).
+A worst-case error of roughly O(εN), average O(εN^^0.5).
+An absolute error of roughly is much less then naive sum of log2s has.
  */
```

Comment:
Thanks!
Naive sum is O(EN^^0.5) on average and O(EN) in the worst-case. But absolute errors are different. I think I can remove all information about O(...).